docker-app/qfieldcloud/core/tests/test_organization.py

```diff
@@ -263,3 +263,48 @@ def _active_users_count(base_date=None):
         )
         # There are still 2 billable users, because self.user3 is staff
         self.assertEqual(_active_users_count(), 2)
+
+    def test_active_users_jobs_deltas_count(self):
+
+        # Let user2 be a member of organization 1
+        OrganizationMember.objects.create(
+            organization=self.organization1,
+            member=self.user2,
+            role=OrganizationMember.Roles.MEMBER,
+        )
+
+        # Idem for user3
+        OrganizationMember.objects.create(
+            organization=self.organization1,
+            member=self.user3,
+            role=OrganizationMember.Roles.MEMBER,
+        )
+
+        # Create a project
+        project1 = Project.objects.create(name="p1", owner=self.organization1)
+
+        # User 2 creates a job
+        Job.objects.create(
+            project=project1,
+            created_by=self.user2,
+        )
+
+        # User 2 creates a delta
+        Delta.objects.create(
+            deltafile_id=uuid.uuid4(),
+            project=project1,
+            content="delta",
+            client_id=uuid.uuid4(),
+            created_by=self.user2,
+        )
+
+        results = self.organization1.list_active_users_jobs_deltas_count()
+        expected = [
+            {
+                "user_id": self.user2.pk,
+                "username": self.user2.username,
+                "jobs_count": 1,
+                "deltas_count": 1,
+            }
+        ]
```

Comment:
In the test also assert the time range that refers to billing period.

Reply:
I am not getting it. I usually use assertions to verify that a certain relation holds between inputs and outputs. What is the input-output relation that you would like me to make an assertion about?